src/ftof.cpp

```diff
@@ -0,0 +1,59 @@
+/*! \file ftof.cpp
+ *
+ * \brief Contains functions to transfer EigenMatrix and EigenVector to FASP
+ *  or to tranfer FASP to Fenics
+ */
+
+#include "ftof.h"
+
+using namespace dolfin;
+
+// Link the dolfin::EigenMatrix Mat_A to dCSRmat format from FASP
+void EigenMatrixTOdCSRmat( dCSRmat* dCSR_A, const EigenMatrix* mat_A)
+{
+  int i,j,k=0;
+  int nrows = mat_A->size(0);
+  int ncols = mat_A->size(1);
+  int nnz = mat_A->nnz();
```

Comment:
Can we make sure that these return ``int`` values?  We may need to cast them
``int nrows = (int) mat_A->size(0)``
etc.
I think they're automatically cast, but it's probably cleaner if we do it ourselves.